go.mod

```diff
@@ -65,6 +65,7 @@ require (
 	go.opentelemetry.io/collector/exporter v0.93.0
 	go.opentelemetry.io/collector/exporter/debugexporter v0.93.0
 	go.opentelemetry.io/collector/exporter/otlpexporter v0.93.0
+	go.opentelemetry.io/collector/featuregate v1.0.1
```

Comment:
Should this version be kept in sync with the other OTel modules?

Reply:
good point